src/web/mjs/connectors/templates/FlatManga.mjs

```diff
@@ -85,7 +85,10 @@ export default class FlatManga extends Connector {
                 try {
                     element.dataset.pagespeedLazySrc = atob(element.dataset.pagespeedLazySrc);
                 } catch(_) { /* ignore */ }
-                return this.getAbsolutePath(element.dataset.src || element.dataset.original || element.dataset.pagespeedLazySrc || element, request.url);
+                try {
+                    element.dataset.aload = atob(element.dataset.aload);
+                } catch(_) { /* ignore */ }
```

Comment:
May use empty `finally` instead of `catch` if not interested in error handling

Reply:
if i use finally it throws an error
https://streamable.com/mto3xk